Apps/Playground/node_modules/@babylonjs/react-native/ios/BabylonNative.cpp

```diff
@@ -38,6 +38,10 @@ namespace Babylon
     Native::Native(facebook::jsi::Runtime* jsiRuntime, void* windowPtr, size_t width, size_t height)
         : m_impl{ std::make_unique<Native::Impl>(jsiRuntime) }
     {
+//        dispatch_sync(dispatch_get_main_queue(), ^{
+//            Plugins::NativeEngine::InitializeGraphics(windowPtr, width, height);            
```

Comment:
?

Reply:
Sorry for the confusion, this is here and commented out because we are supposed to call `InitializeGraphics` from the UI thread, but for some reason it makes the app crash. I added a TODO comment (and this is still an unchecked todo in the GitHub issue for iOS React Native integration). I just need to dig into it more, or get some help from @CedricGuillemet.